python/data_extraction/taxonomy_query.py

```diff
@@ -9,6 +9,9 @@ def __init__(self, key_list=("content_id", "base_path", "title"),
         self.content_store_url = content_store_url
         self.key_list = key_list
 
+    def root_taxon_content_ids(self):
+        return [root_taxon.get('content_id') for root_taxon in self.level_one_taxons()]
```

Comment:
The "level one" naming is being used in part here. I think it would be clearer to use that everywhere, so call this `level_one_taxon_content_ids`. What do you think?

Reply:
Ah, I'm not even sure this function is used. At least it doesn't appear to be used in this commit.

Reply:
removed this